full-stack-tests/presentation/src/learning-snippets/Customization.test.ts

```diff
@@ -0,0 +1,1082 @@
+              groupByLabel: false,
+            }],
+          }, {
+            ruleType: RuleTypes.InstanceLabelOverride,
+            class: { schemaName: "BisCore", className: "SpatialViewDefinition" },
+            values: [{
+              specType: InstanceLabelOverrideValueSpecificationType.Composite,
+              separator: " x ",
+              parts: [
+                { spec: { specType: InstanceLabelOverrideValueSpecificationType.Property, propertyName: "Roll" } },
+                { spec: { specType: InstanceLabelOverrideValueSpecificationType.Property, propertyName: "Pitch" } },
+              ],
+            }],
+          }, {
+            ruleType: RuleTypes.PropertySorting,
+            requiredSchemas: [{ name: "BisCore", minVersion: "1.0.2" }],
+            propertyName: "Pitch",
+          }],
+        };
+        // __PUBLISH_EXTRACT_END__
+
+        const nodes = await Presentation.presentation.getNodes({
+          imodel,
+          rulesetOrId: ruleset,
+        });
+        expect(nodes).to.be.lengthOf(4);
+        expect(nodes[0]).to.containSubset({ label: { displayValue: "-107.42 x -160.99" } });
+        expect(nodes[1]).to.containSubset({ label: { displayValue: "-45.00 x -35.26" } });
+        expect(nodes[2]).to.containSubset({ label: { displayValue: "-90.00 x 0.00" } });
+        expect(nodes[3]).to.containSubset({ label: { displayValue: "0.00 x 90.00" } });
+      });
+
+      it("uses `condition` attribute", async () => {
+        // __PUBLISH_EXTRACT_START__ SortingRule.Condition.Ruleset
+        // The ruleset has root node rule that returns `bis.SpatialViewDefinition` instances with labels
+        // consisting of `Roll` and `Pitch` property values. Also there are customization rule to sort
+        // instances by `Pitch` property if "SORT_INSTANCES" ruleset variable set to "TRUE".
+        const ruleset: Ruleset = {
+          id: "example",
+          rules: [{
+            ruleType: RuleTypes.RootNodes,
+            specifications: [{
+              specType: ChildNodeSpecificationTypes.InstanceNodesOfSpecificClasses,
+              classes: { schemaName: "BisCore", classNames: ["SpatialViewDefinition"] },
+              groupByClass: false,
+              groupByLabel: false,
+            }],
+          }, {
+            ruleType: RuleTypes.InstanceLabelOverride,
+            class: { schemaName: "BisCore", className: "SpatialViewDefinition" },
+            values: [{
+              specType: InstanceLabelOverrideValueSpecificationType.Composite,
+              separator: " x ",
+              parts: [
+                { spec: { specType: InstanceLabelOverrideValueSpecificationType.Property, propertyName: "Roll" } },
+                { spec: { specType: InstanceLabelOverrideValueSpecificationType.Property, propertyName: "Pitch" } },
+              ],
+            }],
+          }, {
+            ruleType: RuleTypes.PropertySorting,
+            condition: "GetVariableBoolValue(\"SORT_INSTANCES\")",
```

Comment:
Can we use plain `true` and `false` literals instead of ruleset variables?

Reply:
Changed to plain literals instead of ruleset variable